include/api.php

```diff
@@ -527,6 +527,7 @@ function api_get_user(App $a, $contact_id = null)
 
 	// Searching for contact URL
 	if (!is_null($contact_id) && (intval($contact_id) == 0)) {
+		/// @todo nupplaphil 20190119 normalizeLink of id 0 - why?
 		$user = DBA::escape(Strings::normaliseLink($contact_id));
```

Comment:
I guess it can not only be an ID but also an URL. There are ugly things there in the api.php.

Reply:
but how can `$contact_id` be an URL?
-> We are checking it with `intval($contact_id) == 0`

Reply:
Yeah. When an ```intval``` returns ```0``` - but isn't null - then it possibly contains an URL.

And yes, this is ugly.

Reply:
Ah :D OK at least I understand it. I'll remove the hint :-)